docs/book/beyond_basics/what_is_a_webhook.md

```diff
@@ -0,0 +1,81 @@
+# Webhook
+
+Webhooks are HTTP callbacks, providing a way for notifications to be delivered to an external web server.
+A web application implementing webhooks will send an HTTP request (typically POST) to other application when certain event happens. 
+In kubernetes world, there are 3 kinds of webhooks: admission webhook, authorization webhook and CRD conversion webhook.
+
+## Admission Webhook
+
+Admission webhooks are HTTP callbacks that receive admission requests, process them and return admission responses.
+There are two types of admission webhooks: mutating admission webhook and validating admission webhook.
+With mutating admission webhooks, you may change the requests to enforce custom policy, e.g. injecting a sidecar container
```

Comment:
`With mutating admission webhooks, you may change the request object before it is stored (e.g. for performing defaulting of fields or enforcing a custom policy)